src/README.md

```diff
@@ -0,0 +1,51 @@
+# Editorial principles
+
+## Who is this book for?
+
+## Contents of sections
+
+### Introduction
+
+> If I know nothing about the topic and read this paragraph I will know a
+> surface description of the problem type and the name of the solution approach.
+
+### Typical Problem
+
+> A brief (but comprehensive) description of a typical problem. If I do not know
+> anything about the subject matter I can understand the problem and the problem
+> parameters. If I **do** know the subject matter I should be able to solve the
+> problem.
+
+### Theory
+
+- Discuss some theoretic principles (some basic technical language is acceptable)
+- Use a smaller example (different to the typical problem) that can be done by
+  hand.
+- Some theoretic concepts might be illustrated using the typical problem.
+
+### Code
+
+- Give a brief (blog post style) overview of using each langauge to solve the
+problem.
+- A functional approach is used: everything is embedded in functions that are
```

Comment:
The word "functional" is incorrect here: that would actually imply a completely different paradigm and set of languages. I mean: use functions.

Reply:
Put the triangle in.